packages/grid/x-data-grid/src/models/params/gridEditCellParams.ts

```diff
@@ -45,6 +46,7 @@ export interface GridCellEditStartParams<R extends GridValidRowModel = any, V =
   reason?: GridCellEditStartReasons;
   /**
    * If the reason is related to a keyboard event, it contains which key was pressed.
+   * @deprecated No longer needed.
```

Comment:
We don't need it, but it could still make sense to pass it since the [`cellEditStart` is a public event](https://mui.com/x/react-data-grid/events/)